src/app/archive/README.md

```diff
@@ -0,0 +1,87 @@
+Archive
+=======
+
+The Mina daemon does not remember the entire history of the blockchain.
+On the contrary, it only remembers a couple of blocks backwards, called
+the *transition frontier*. If storing historic transaction data is
+desired, this Archive needs to be set up next to the daemon itself.
+
+Prerequisites
+-------------
+
+The Archive stores its data in a PostgreSQL database, so its necessary
+to set one up before proceeding to run the Archive. The way one
+installs Postgres software depends the operating system. However, in
+some setups it might be more convenient to use the [official Postgres
+Docker image](https://hub.docker.com/_/postgres) instead. In that case
+the following command will set the database up:
+
+```shell
+$ docker run -d --name pg-mina-archive \
+    -p 5432:5432 \
+    -e POSTGRES_PASSWORD='*******' \
+    -e POSTGRES_HOST_AUTH_METHOD=trust \
+    -e POSTGRES_DB=mina_archive \
+    -e POSTGRES_USER=pguser \
+    postgres:latest
+```
+
+Note that setting the authentication method to `trust` is very unsafe,
+because it allows anyone to connect without giving a password. While
+convenient and acceptable in development settings, this option should
+never be used in production. Also note that even with authentication
+method set to `trues`, its still necessary to provide a password for
```

Comment:
is `trues` correct? :D

Reply:
Obviously it should spell `trust`. Thanks for spotting that!

Reply:
Fixed.